### PR TITLE
Add info about Delete Operation into README

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Part of #102 

Added info about `Delete` operation, improved README

@nikitin-da PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,29 +15,15 @@
...
-  .get()
-  .cursor()
-  .withQuery(new Query.Builder()
-    .table("tweets")
-    .build())
-  .prepare()
-  .executeAsBlocking();
-```
-
+####1. Get Operation
```
Comment (nikitin-da) [inline README.md]: What about 2 and 3?)

Comment (artem-zinnatullin) [inline README.md]: Damn, fixing...

Comment (artem-zinnatullin) [inline README.md]: Fixed!

**Comment by nikitin-da** [comment]
a way better=)

**Pull request merged**